compose/install/index.md

```diff
@@ -0,0 +1,39 @@
+---
+description: How to install Docker Compose
+keywords: compose, orchestration, install, installation, docker, documentation
+title: Install Docker Compose
+toc_max: 2
+---
+
+This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.
+
+
+## Prerequisites
+
+* Docker Compose requires Docker Engine.
+* Docker Compose plugin requires Docker CLI.
+
+## Install Compose
+
+Check what installation scenario fits your needs. Are you looking to:
+
+* __Get latest Compose and pre-requirements of the bat__:
+Install Docker Desktop for your platform. This is the fastest route and you get Docker Engine, Docker CLI with the Compose plugin. 
+Docker Desktop is available for Mac, Windows and Linux.
+
+* __Install Compose plug in:__
+  + __(Mac, Win, Linux) Docker Desktop__: If you got Desktop installed then you're all set, no need to install anything separatedly.
```

Comment:
```suggestion
  + __(Mac, Win, Linux) Docker Desktop__: If you have Docker Desktop installed then you do not need to install anything separately.
```

Reply:
POV: Thinking that I was going for listing actions, but one could have done this at an earlier stage and taking your suggestion I think something "If you have (already) installed DD..." to work better, yes. Thanks.

Spelling: Thanks. Was thinking about this. Grammarly was my go-to. Is there a spelling extension for VC? If any exist (I'll have look) as well) that we're using as team that you know of at least.

Tone: I find the suggestion to replace "then you're all set" to "then you do not need" is one for a more neutral and dry tone of voice. 
I am aiming to, when writing, to use direct, second-person pronouns (such as “you”) and simple, casual phrases that create a friendly and inviting tone.  I'd like to advocate for that being aligned with our developers mostly informal culture, our visual identity, specifically our mascots.

Reply:
Thanks for the review @aevesdocker. I will wait to see if have more feedback next week, might commit some of your suggestions on Monday and then ping after tech review or time cut off (?).

Reply:
PS: Changed this as well.